### PR TITLE
add mariadb to compatibility

### PR DESCRIPTION
**Pull request by ZStriker19**
### What does this PR do?
After the release of a ddtrace with the mariadb integration these docs should be merged to include supported mariadb versions and link to docs.

### Motivation
This branch: https://github.com/DataDog/dd-trace-py/tree/add_mariadb

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
Hi @ZStriker19, I added the Do Not Merge label to this one since it depends on a release before it can be merged. Will you please remove this label once the release goes out so that we know it's ready for our review and merge?

**Comment by apigirl** [comment]
I am going to merge this since the related PR adding the functionality is live.

**Review by apigirl — APPROVED** [review]

**Pull request merged**
